### ISSUE TITLE
As a keyboard or screen user, I want to navigate the timetree without a mouse or touch device so that I can explore the content in multiple paths and access all the information in a manner that is equivalent to the experience of a sighted  user.

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/lenape-timetree#65]
## dev notes
- [x] make leaves keyboard navigable
- [x] make keyboard nav order logical
- [x] make the leaves buttons (role = button)
  - [x] aria label: display title of the leaf
  - [x] area described by: display date and branch
   - [x] trigger button on enter & space bar; transfer focus in, then close button transfers focus back. 
- [x] disable leaf button when tag is active and leaf does not have tag
- [x] make the panel act like a modal (transfer focus; announce; for keyboard user, send them to the close button after tabbing through last tag; return focus to the next leaf after the clicked one)
- [x] separate tag from panel: closing panel should no longer close the tag
- [x] h2 equivalent label for each branch (possible target from the branch legend in the intro)
- [x] make the footer tab order first before the tree
- [x] make the intro dom order first before the tree
- ~allow zoom on desktop~  (tracked separately)
- ~buttons for zoom in/out~  (tracked separately)
- [x] #199
- [x] ensure visually hidden content is also hidden from screen readers
- [x] ensure visually hidden/disabled buttons are disabled or hidden from screen readers

---
revisions after testing:
- [x] update node order in the dom to follow visual left-to-right branch order
- [x] revert focus style customization for active tag close button
- [x] ensure invisible branch headers retain `tabindex="-1"` (focussable but not part of default tab sequence)
- [x] clean up focus style for invisible branch header when focused
- [x] when tag is active, make tagging act like a modal (contain within tree + tag)

**Comment by rlskoeser** [comment]
implemented and tested on #202
